lib/desired-caps.js

```diff
@@ -192,6 +192,9 @@ let desiredCapConstraints = _.defaults({
   includeSafariInWebviews: {
     isBoolean: true
   },
+  additionalWebviewBundleIds: {
+    isArray: true
```

Comment:
I'm thinking about making it a comma separated string instead. I see people sometimes having problems poviding arrays to caps, especially in Java